firewall/exemption.go

```diff
@@ -0,0 +1,30 @@
+package firewall
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+)
+
+type Exemption struct {
+	Destination net.IP
+	Port        int
```

Comment:
Is this dest port? I guess for now there is no good way to have multiple destination ports in Connect so seems OK but we could generalise this to non-connect workloads too and then you might want to allow traffic to one port but not another on the source?